src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -302,9 +305,18 @@ async def get_feed_tip(query_id: bytes, autopay: TellorFlexAutopayContract) -> A
         msg = "can't suggest feed, autopay contract not connected"
         error_status(note=msg, log=logger.critical)
         return None
-    single_query = {query_id: CATALOG_QUERY_IDS[query_id]}
+    try:
+        single_query = {query_id: CATALOG_QUERY_IDS[query_id]}
+    except KeyError:
+        CATALOG_QUERY_IDS[query_id] = query_id.hex()
```

Comment:
If it's not in `CATALOG_QUERY_IDS`, then wouldn't the reporter not be able to report for it? Since it's an unsupported queryId?